tcl/handlers.tcl

```diff
@@ -3,14 +3,14 @@ namespace eval qc::handlers {
     namespace export call exists validate2model validation
     namespace ensemble create
 
-    proc call {path method} {
+    proc call {method path} {
```

Comment:
It's possible I think. Here's a breakdown of how each pattern is checked:

* Check that all colon variables in the pattern are present in the argument names.
* Check the number of parts from both pattern and path match and the number of arguments for the handler is not less than the number of colon variables in the pattern.
* Check each part of the pattern matches the corresponding path part and substitute colon variables in the pattern for their values from the path. 
* If the parts of the substituted path when put together match the pattern:
  * make the args
  * build the command
  * set the command_dict accordingly

I'll take a look and see how many dependent variables there are and how easy it might be to break it up.